newrelic_lambda_cli/constants.py

```diff
@@ -0,0 +1,5 @@
+from enum import Enum
+
+
+class Constants(Enum):
```

Comment:
This also doesn't look like an enum to me. Can probably just remove the enum altogether?

Reply:
This could be refactored to perhaps a named tuple or a global variable or something else - I'll look into making some changes, since the Constants class itself never really get's iterated over or a lot of other things